add docs for formsort (#4234)
segmentio/segment-docs: src/connections/sources/catalog/cloud-apps/formsort/index.md

---
title: "Formsort Source"
id: f02zgzAWMG
hidden: true
---

[Formsort](https://formsort.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank”} is a low code form builder empowering product and growth teams to build and optimize highly complex, customizable flows. Increase form conversion by over 20% - without the dev time.

This source is maintained by Formsort. For any issues with the source, [contact the Formsort support team](mailto:support@formsort.com).

## Getting Started

1. From your Segment UI’s Sources page click on “Add Source”.
2. Search for “Formsort” within the Sources Catalog and confirm by clicking “Connect”.

> 3. Give the source a nickname and follow the set up flow to “Add Source”. The nickname is a label used in the Segment interface, and Segment creates a related schema name which you query against in your warehouse. The nickname can be whatever you like, but we recommend sticking to something that reflects the source itself and distinguishes amongst your environments (Eg. formsort-prod or formsort-dev).

Reviewer: ```suggestion
3. Give the source a nickname and follow the set up flow to “Add Source”. The nickname is a label used in the Segment interface, and Segment creates a related schema name which you query against in your warehouse. The nickname can be whatever you like, but Segment recommends sticking to something that reflects the source itself and distinguishes amongst your environments (Eg. formsort-prod or formsort-dev).
```

---
title: "Formsort Source"
id: f02zgzAWMG
hidden: true
---

[Formsort](https://formsort.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners) is a low code form builder empowering product and growth teams to build and optimize highly complex, customizable flows. Increase form conversion by over 20% - without the dev time.

This is an [Event Cloud Source](https://segment.com/docs/sources/#event-cloud-sources) which can not only export data into your Segment warehouse, but they can also federate the exported data into your other enabled Segment Destinations.

> This source is maintained by Formsort. For any issues with the source, [contact their Support team](mailto:support@formsort.com).

Reviewer: ```suggestion
This source is maintained by Formsort. For any issues with the source, [contact the Formsort support team](mailto:support@formsort.com).
```

## Getting Started

1. From your Segment UI’s Sources page click on “Add Source”.
2. Search for “Formsort” within the Sources Catalog and confirm by clicking “Connect”.

> 3. Give the Source a nickname and follow the set up flow to “Add Source”. The nickname is a label used in the Segment interface, and Segment creates a related schema name which you query against in your warehouse. The nickname can be whatever you like, but we recommend sticking to something that reflects the source itself and distinguishes amongst your environments (Eg. formsort-prod or formsort-dev).

Reviewer: Current guidance from content team is that "sources" and "destinations" aren't capitalized

```suggestion
3. Give the source a nickname and follow the set up flow to “Add Source”. The nickname is a label used in the Segment interface, and Segment creates a related schema name which you query against in your warehouse. The nickname can be whatever you like, but we recommend sticking to something that reflects the source itself and distinguishes amongst your environments (Eg. formsort-prod or formsort-dev).
```

| `event`         | Flow event type                 |
| `flowLabel`     | Label of the flow               |
| `variantLabel`  | Label of the variant            |
| `variantUuid`   | UUID of the variant             |
| `responderUuid` | UUID of the responder           |
| `userId`        | Identifying id for the responder | 

This table lists event-specific properties Formsort sends to Segment:

| Property Name | Description                                                                         |
| ------------- | ----------------------------------------------------------------------------------- |
| stepId        | For step events, the name of the step                                               |
| stepIndex     | For step events, the index of the step in the flow.                                 |
| msSpentOnStep | For StepCompleted events, the time in milliseconds the responder spent on the step. |

Optionally, form answers submitted by responders can also be included in events sent to Segment. If enabled, responder answers will appear in an `answers` object for each event.

## Adding Destinations

Now that your Source is set up, you can connect it with Destinations.

> Log into your downstream tools and check to see that your events appear as expected, and that they contain all of the properties you expect. If your events and properties don’t appear, check the [Event Delivery](https://segment.com/docs/connections/event-delivery/) tool, and refer to the Destination docs for each tool for troubleshooting.

Reviewer: ```suggestion
Log into your downstream tools and check to see that your events appear as expected, and that they contain all of the properties you expect. If your events and properties don’t appear, check the [Event Delivery](/docs/connections/event-delivery/) tool, and refer to the Destination docs for each tool for troubleshooting.
```
